src/Native/Runtime/unix/UnwindHelpers.cpp

```diff
@@ -16,7 +18,9 @@
 #include "UnwindHelpers.h"
 
 // libunwind headers
+#ifndef TARGET_64
 #include <libunwind.h>
```

Comment:
Can we also stop building and linking libunwind ?

Reply:
Unfortunately not. It's still used to find the LSDA as I haven't touched this part.

Reply:
Does LDSA differ by architecture or is it same for aarch64 as https://itanium-cxx-abi.github.io/cxx-abi/exceptions.pdf?

Reply:
The content of the LSDA is CoreRT specific. What we are still need libunwind for is to find the LSDA for a memory address. We can replace this part of the code, too. But to work with different linker the code would be mostly the same as what we have with libunwind now. 